libbeat/publisher/includes/includes.go

```diff
@@ -0,0 +1,15 @@
+package includes
+
+import (
+	// load supported output plugins
```

Comment:
we could generate this in the future ... but as we are not going to add outputs ...